### PR TITLE
4296 redirect save paper petition

### PR DESCRIPTION
**Pull request by mrinsin**
Redirecting to my document qc inbox after saving a case for later

### CODE REVIEW

File: web-client/src/presenter/actions/clearAlertsAction.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ import { state } from 'cerebral';
  */
 export const clearAlertsAction = ({ get, props, store }) => {
   const saveAlerts = get(state.saveAlertsForNavigation);
+  console.log(saveAlerts);
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/clearAlertsAction.js]: Remove please

File: web-client/src/presenter/actions/setSaveAlertsForNavigationAction.js [hunk 2]
```diff
@@ -7,5 +7,6 @@ import { state } from 'cerebral';
  * @param {Function} providers.store the cerebral store used for setting the state.saveAlertsForNavigation
  */
 export const setSaveAlertsForNavigationAction = ({ store }) => {
+  console.log('setting save alerts');
```
Comment (rachaelparris) [inline web-client/src/presenter/actions/setSaveAlertsForNavigationAction.js]: Remove please

File: web-client/src/presenter/sequences/navigateBackFromPrintPreviewSequence.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// import { clearAlertsAction } from '../actions/clearAlertsAction';
```
Comment (rachaelparris) [inline web-client/src/presenter/sequences/navigateBackFromPrintPreviewSequence.js]: Could you remove the things that are commented out please?

File: web-client/src/presenter/sequences/navigateBackFromPrintPreviewSequence.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import { navigateBackAction } from '../actions/navigateBackAction';
+import { navigateToPathAction } from '../actions/navigateToPathAction';
+// import { setSaveAlertsForNavigationAction } from '../actions/setSaveAlertsForNavigationAction';
+import { stopShowValidationAction } from '../actions/stopShowValidationAction';
+
+export const navigateBackFromPrintPreviewSequence = [
+  // setSaveAlertsForNavigationAction,
+  // clearAlertsAction,
+  stopShowValidationAction,
+  getHasAlternateBackLocationAction,
```
Comment (rachaelparris) [inline web-client/src/presenter/sequences/navigateBackFromPrintPreviewSequence.js]: Where is `state.screenMetadata.backLocation` that is being used in this action being set?

File: web-client/src/presenter/actions/clearAlertsAction.js (hunk 1 above)
Comment (mrinsin) [reply web-client/src/presenter/actions/clearAlertsAction.js]: Thanks!

File: web-client/src/presenter/sequences/navigateBackFromPrintPreviewSequence.js (hunk 3 above)
Comment (mrinsin) [reply web-client/src/presenter/sequences/navigateBackFromPrintPreviewSequence.js]: Thanks! Looks like I forgot to clean up old code!

File: web-client/src/presenter/sequences/navigateBackFromPrintPreviewSequence.js (hunk 4 above)
Comment (mrinsin) [reply web-client/src/presenter/sequences/navigateBackFromPrintPreviewSequence.js]: Not entirely sure, this was code that existed before - but this was all code I was working on for an issue we decided not to proceed with (and I forgot to clean up) so I will be reverting all these files to their original state!

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**